src/core/models/filter.js

```diff
@@ -192,6 +192,13 @@ export default class Filter {
     return Filter._fromMatcher('builtin.location', '$near', { lat, lng, radius });
   }
 
+  /**
+   * Create a new RADIUS_FILTER.
+   * @param {number} radius
+   */
+  static locationRadius (radius) {
```

Comment:
Can we remove this static method?

Reply:
sorry I must have mouse slipped or something